src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+);
```

Comment:
- [ ] Kindly make this component a class-based component. For this project, your components are required to be class-based components, and later you'll have to refactor them to functional hooks. 

```javascript
import React, { Component } from 'react';
import Calculator from './components/Calculator';

class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

Reply:
Thanks. Changed to class based.
![image](https://user-images.githubusercontent.com/13853450/176790191-1cc3897b-f50a-410c-a074-e88ef674b009.png)

Reply:
Thanks for your review. Done with changes.